admin/dist/mac/Standard.plist

```diff
@@ -32,6 +32,8 @@
 	<string>AppleScriptSuite.sdef</string>
 	<key>NSAppleScriptEnabled</key>
 	<true/>
+	<key>NSAppleEventsUsageDescription</key>
+	<string>Allow Last.fm to listen for AppleEvents.</string>
```

Comment:
Is this displayed to the user? If it is, we should put something more along the lines of "Allow Last.fm to see what track is currently playing in iTunes or Apple Music"

Reply:
This is displayed to the user yes. That message will also appear for the Spotify permissions pop-up too unfortunately. Makes it harder to come up with a generic message
See the last few comments on here - https://jira.last.fm/browse/DATA-13127

Reply:
My suggestion is now: "**For Music and iTunes, this enables scrobbling. For Spotify, this lets you see Last.fm info about the currently playing song**"